src/views/Player.vue

```diff
@@ -89,7 +89,8 @@ export default {
       isBrowserSupported: true,
       browserErrorHandlingValue: {
         'failsafeType': 'g-form',
-        'failsafeUrl': ''
+        'failsafeUrl': '',
+        'youtubeId': ''
```

Comment:
Added 'youtubeId' here because if there's a g-form, there will always be a youtube video to be seen